.travis.yml

```diff
@@ -43,6 +43,7 @@ before_script:
 
 script:
 - ./gradlew build connectedCheck test jacocoTestReport
+- ./validate-play-store-description.sh
```

Comment:
Not sure which stage is the best for the validation

Reply:
Instead of the earlier `chmod` you could simpler add "bash" here